Sources/CryptoSwift/Salsa20.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  Salsa20.swift
+//  CryptoSwift
+//
+//  Created by Dennis Michaelis on 27.03.16.
+//  Copyright © 2016 Dennis Michaelis. All rights reserved.
+//
+
+final public class Salsa20 {
```

Comment:
maybe the ChaCha20 could be extend from Salsa20, but the Vector initialization is different, some constants like "expand 32-byte k" can be reused.